### DISCUSSION TITLE
Daily 18/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#86] [Daily Sprint]
- Que hice ayer?
- Que haré hoy?
- Problemas o Impedimentos.

**Comment by rpgrca** [comment]
Ayer terminé de hacer el esqueleto de la lógica del back-end y pushee la rama a Github. También tuve que ir a la oficina así que me quedé con poco tiempo para trabajar y no pude hacer el diagrama de caso de uso, no sé tampoco cuál tomó @Brian-Almada para hacer o si iba a hacer todo, averigüaré y haré alguno que no vaya a hacer

**Comment by NataliaAlvarezIspc** [comment]
Ayer terminé con el diagrama entidad-relación y lo subí a la wiki. También estuve ordenando las historias de usuarios y empecé a agregarles las tareas correspondientes. Además estuve completando algo de la documentación en la wiki.
Hoy estaré con el diagrama de casos de uso general y continuaré agregando tareas a las historias de usuario. Por otro lado quiero terminar el capítulo de Angular de la plataforma, aunque no sé si llegaré con el tiempo.

**Comment by BraianTroncoso** [comment]
Ayer terminé unas clases de back-end en python y js(Funciones y objetos), y me puse con la BDD a modificarla.
También leí algo de documentación pero ya estaba cansado.
Hoy posiblemente siga repasando unas clases de back y vea algo de Angular.

**Comment by Brian-Almada** [comment]
Ayer estuve viendo tutoriales para saber como armar el diagrama de casos de uso y armando el borrador. Pero veo que ya hicieron esa tarea, así que continuaré hoy con Angular que lo dejé en pausa.
